database/schema.sql

```diff
@@ -18,6 +17,15 @@ create table "messages" (
   "messageId" serial,
   "message"   text    not null,
   "chatId"    text    not null,
+  "sender"    text    not null,
   "createdAt" timestamptz(6) not null default now(),
   primary key ("messageId")
 );
+
+create table "users" (
+  "userId" serial,
+  "userName"  text not null,
+  "chatRooms" text not null,
```

Comment:
Postgres has a `json` data type. If you switch to that, then you won't have to `JSON.parse` the `chatRooms`.